repose-aggregator/components/services/container-configuration-service/impl/src/main/scala/org/openrepose/nodeservice/containerconfiguration/DeploymentConfigPatchUtil.scala

```diff
@@ -148,6 +156,22 @@ object DeploymentConfigPatchUtil {
     base
   }
 
+  /**
+    * This function does not return a copy of the base configuration, but instead mutates it directly.
+    *
+    * @param base  the base configuration object
+    * @param patch the patch configuration object, the values of which will override the base
+    * @return the base configuration object with the patch applied
+    */
+  def patchViaHeader(base: ViaHeader,
+                            patch: ViaHeaderPatch): ViaHeader = {
```

Comment:
I have no idea what happened on that one, because the formatter fixed it as expected. Weird.